src/clj/battlebots/arena/generation.clj

```diff
@@ -11,7 +11,7 @@
 (defn- pos-open
   "returns true of false depending if a given coodinate in a given arena is open"
   [[x y] arena]
-  (= (:open arena-key) (get-in arena [x y])))
+  (= (:open arena-key) (get-in arena [y x])))
```

Comment:
Yeah I agree with this. The problem was early on we had two different interpretations floating around, and at the time it seemed easier to refactor to the current interpretation. Once I get all the tests in place I'm going to put some research in to the pros and cons of both structures and I'll make a effort to go back and refactor.